packages/codemods/src/transforms/v4/web-react/__testfixtures__/unstable-slider-component-name.input.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+// @ts-ignore: No declaration -- The library is not installed; we don't need to install it for fixtures.
+import { UNSTABLE_Slider, UNSTABLE_UncontrolledSlider } from '@lmc-eu/spirit-web-react';
```

Comment:
```suggestion
import { UNSTABLE_Slider } from '@lmc-eu/spirit-web-react/components/UNSTABLE_Slider';
import { UNSTABLE_UncontrolledSlider } from '@lmc-eu/spirit-web-react/components/UNSTABLE_UncontrolledSlider';
```
will it work with this too?

Reply:
it will work, but why do you think it is better?

Reply:
I think the question is asked because the developer is free to import the components even this way, so we should be sure that this will work.

Reply:
Hmm, I see what you mean, and I have to correct myself—it will not work with the longer import path. 
But I will make some updates to make it work.